src/System/TimedScopes/Correlation.cs

```diff
@@ -0,0 +1,302 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#region Using Directives
```

Comment:
It would be nice to remove regions for using directives

Reply:
I removed them from TimedScopes classes, should I do the same in other projects?